content/app/github/wrappers/line-element-wrapper.js

```diff
@@ -11,9 +11,46 @@ define("content/app/github/wrappers/line-element-wrapper", function(require, exp
 		$statics : {
```

Comment:
Yes, you have right here regarding making 2 separated classes. But I need to think little about further architecture before I make changes...
I'll try to continue tomorrow with this pull req.